release-notes/5.0/preview/api-diff/rc1/.Net/5.0-rc1_System.Buffers.md

```diff
@@ -0,0 +1,14 @@
+# System.Buffers
+
+``` diff
+ namespace System.Buffers {
+     public readonly struct ReadOnlySequence<T> {
++        public long GetOffset(SequencePosition position);
+     }
+     public ref struct SequenceReader<T> where T : struct, IEquatable<T> {
+-        public bool TryReadTo(out ReadOnlySpan<T> sequence, ReadOnlySpan<T> delimiter, bool advancePastDelimiter = true);
```

Comment:
Did we remove something here?

Reply:
the param was renamed from `sequence` to `span`